### DISCUSSION TITLE
How to start a exe with a space path by shell Command?

### DISCUSSION THREAD

**Discussion by happymingjieli** [tauri-apps/tauri#5387] [Q&A]
### Describe the bug

 (on Windows) I am trying to use run to execute programs that have spaces in the pathname and take command line arguments. Something like the following example:

   const child = await new Command('cmd', [
      '/C',
      `start C:/Program Files/path/FaBuTing_Btest.exe -arg`,
    ]).spawn();

but it doesnt work because of the space in the path (Program Files), so what is the correct way to deal with it?

### Platform and versions

![image](https://user-images.githubusercontent.com/49396751/195076611-d17c448a-fbfa-49ec-ac79-b45090038afd.png)

**Comment by amrbashir** [comment]
this should work
```js
const child = await new Command('cmd', [
      '/C',
      'start',
      'C:\\Program Files\\path\\FaBuTing_Btest.exe`,
      '-arg'
    ]).spawn();
```

**Reply by happymingjieli** [reply]
> this should work
> 
> ```js
> const child = await new Command('cmd', [
>       '/C',
>       'start',
>       'C:\\Program Files\\path\\FaBuTing_Btest.exe`,
>       '-arg'
>     ]).spawn();
> ```

thanks for your answer, I write as below, but it does not work

    const child = await new Command('cmd', [
      '/C',
      'start',
      'C:\\Program Files\\path\\FaBuTing_Btest.exe',
      '-RCWebControlEnable',
      '-RCWebInterfaceEnable',
      '-RenderOffscreen',
    ]).spawn();

![1665544886873_11D2A017-97E1-4d8b-8831-9D86EA3EED8F](https://user-images.githubusercontent.com/49396751/195242284-844cb428-f9ad-4ef4-98aa-dea8af26ca81.png)
